accepted/2021/statics-in-interfaces/README.md

```diff
@@ -837,6 +839,20 @@ namespace System
         static abstract TSelf RotateRight(TSelf value, TSelf rotateAmount);
 
         static abstract TSelf TrailingZeroCount(TSelf value);
+
+        // These methods allow getting the underlying bytes that represent the binary integer
+
+        void CopyTo(Span<byte> destination);
+
+        void CopyTo(byte[] destination);
+
+        void CopyTo(byte[] destination, int startIndex);
+
+        nuint GetByteCount();
+
+        nuint GetBitLength();
```

Comment:
Why these methods return `nuint` instead of a basic `int` or `uint` ?
What does `Bit Length` means ? Is it equals to `GetByteCount() * 8` ?
And why `Byte` is `Count` but `Bit` is `Length` ?

Reply:
They were updated to return `int` in API review. `nuint` is the "most correct type" where-as `int` is the .NET standard type and we briefly covered the nuances and limitations of each.

> And why Byte is Count but Bit is Length ?

That's just the existing convention we had and soo it was followed for consistency.